.circleci/config.yml

```diff
@@ -19,11 +19,11 @@ commands:
             BREW_NO_UPDATE=1 ./opt/system-setup.py
             # git clone git://github.com/antirez/redis.git --branch 5.0.7; (cd redis; make malloc=libc -j $(nproc); make install); redis-server --version
             ./get_deps.sh cpu
-      - save_cache:
-          paths:
-            - deps
-            - /usr/local
-          key: build-dependencies-{{ checksum "get_deps.sh" }}
+      # - save_cache:
```

Comment:
I disabled it for safe guarding. After this test passess will push a new one enabling caching again to double check

Reply:
checking on this new commit if CI with caching passes the coverage test

Reply:
```
Found a cache from build 2503 at build-dependencies-g56SbIkKcrU1pwadgbiX37Oa8YFgTUciKupRTnuyJQY=
Size: 472 MiB
Cached paths:
  * /root/project/deps
  * /usr/local

Downloading cache archive...
Validating cache...

Unarchiving cache...
```
@lantiga working as expected. But I disabled the fallback to non exact dependencies. On the case of the checksum not being exact it will download new ones ( so that we can change versions and not break CI )